devtools/aptos-cargo-cli/src/lib.rs

```diff
@@ -19,19 +19,28 @@ const APTOS_CLI_PACKAGE_NAME: &str = "aptos";
 // Note: these paths should be relative to the root of the `aptos-core` repository,
 // and will be transformed into UTF-8 paths for cross-platform compatibility.
 const RELEVANT_FILE_PATHS_FOR_COMPILER_V2: [&str; 7] = [
+    ".github/actions/move-tests-compiler-v2",
+    ".github/workflows/move-test-compiler-v2.yaml",
     "aptos-move/aptos-transactional-test-harness",
     "aptos-move/e2e-move-tests",
     "aptos-move/framework",
     "aptos-move/move-examples",
     "third_party/move",
-    ".github/workflows/move-test-compiler-v2.yaml",
-    ".github/actions/move-tests-compiler-v2",
+];
+const RELEVANT_FILE_PATHS_FOR_EXECUTION_PERFORMANCE_TESTS: [&str; 5] = [
+    ".github/workflows/execution-performance.yaml",
+    ".github/workflows/workflow-run-execution-performance.yaml",
```

Comment:
do these matter, since they are not applied on the PR itself, but pulled from main anyways ? (as far as I understand)

Reply:
> do these matter, since they are not applied on the PR itself, but pulled from main anyways ? (as far as I understand)

If you're testing a change locally, you can temporarily change the workflow to use `pull_request` so that it runs whatever your code is in the PR. I've temporarily done it in this PR: https://github.com/aptos-labs/aptos-core/pull/13170/files#diff-8b02be56513144a070e205ed7fcd61b9162d665d5f451afbe80884768b62fe87R26.

But, overall, it's probably not a super big deal. Just thought it nice to trigger the job if we change the workflow 😄

Reply:
Aah, I was wrong 😄 If you look at the code, it already uses `pull_request` as the target for the workflow. So, if you change it, and it triggers, it'll already do the correct thing 😄 See: https://github.com/aptos-labs/aptos-core/blob/346b611c8903fdf9b095726f1bafe0eb2c724fc9/.github/workflows/execution-performance.yaml#L4